File: js/odd_or_even.js
function oddOrEven(array) {
   // .reduce() arr
   // if arr % 2 === 0, then return 'even'
   // else if arr % 2 !== 0 then return 'odd'
   // else if arr is empty, return 'even'

   const sum = array.reduce((accumulator, currentValue) => accumulator + currentValue, 0)

   if (sum % 2 === 0) {
     return 'even'
   } else if (sum % 2 !== 0) {
     return 'odd'
   } else {
     return 'even'
   }
}

File: java/odd_or_even.java
import java.util.Arrays;

public class Codewars {
  public static String oddOrEven (int[] array) {

    // sum of all numbers in array, using built in stream method
    int sum = Arrays.stream(array).sum();

    // tests for even first
    if (sum % 2 == 0) {
       return "even";
     }
     // if doesn't pass above test, num is odd
     return "odd";
  }
}
